src/main/java/core/basesyntax/UserService.java

```diff
@@ -1,8 +1,16 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.UserNotFoundException;
+
 public class UserService {
 
     public int getUserScore(String[] records, String email) {
-        return 0;
+        for (String users : records) {
+            String[] userData = users.split(":");
+            if (userData.length == 2 && userData[0].equals(email)) {
```

Comment:
```suggestion
            if (userData[0].equals(email)) {
```
we can be sure that all data will have this form

Reply:
You mean equals?If Null data will NPE.about this?